target/cheri-common/op_helper_cheri_common.c

```diff
@@ -880,6 +880,15 @@ target_ulong CHERI_HELPER_IMPL(ctestsubset(CPUArchState *env, uint32_t cb,
     return (target_ulong)is_subset;
 }
 
+target_ulong CHERI_HELPER_IMPL(cseqx(CPUArchState *env, uint32_t cb,
+                                     uint32_t ct))
+{
+    const cap_register_t *cbp = get_readonly_capreg(env, cb);
+    const cap_register_t *ctp = get_readonly_capreg(env, ct);
+
+    return cap_exactly_equal(cbp, ctp);
+}
```

Comment:
Hm. I don't know if this should live here or in target/riscv, given target/mips has its own cexeq instead...

Reply:
Since the actual comparison has been factored out, I'd move this to the RISCV helpers.

Reply:
I'm tempted to make `target/mips` translate `cexeq` to `cseqx`, since it's the same code?

Reply:
Yes that would probably be even better. I guess we could de-Riscv-ify the name of the helper though. I prefer readable names to weird abbreviations.

Reply:
Ah but that would make the translate code awkward. Let's just keep the RISCV name then